rubin_sim/maf/metrics/transient_metrics.py

```diff
@@ -154,15 +162,18 @@ def light_curve(self, time, filters):
         return lc_mags
 
     def run(self, data_slice, slice_point=None):
-        """ "
-        Calculate the detectability of a transient with the specified lightcurve.
+        """
+        Calculate the detectability of a transient with the specified
+        lightcurve.
 
         Parameters
         ----------
         data_slice : numpy.array
-            Numpy structured array containing the data related to the visits provided by the slicer.
+                     Numpy structured array containing the data related to the
```

Comment:
This is the wrong level of indent I think -- only four spaces indent (and not tabs) for parameter descriptions.